mvrx/src/main/kotlin/com/airbnb/mvrx/MavericksView.kt

```diff
@@ -295,4 +300,31 @@ interface MavericksView : LifecycleOwner {
         onFail: (suspend (Throwable) -> Unit)? = null,
         onSuccess: (suspend (T) -> Unit)? = null
     ) = _internalSF(subscriptionLifecycleOwner, asyncProp, deliveryMode, onFail, onSuccess)
+
+    /**
+     * Subscribes to on the given Flow. This can be utilized to create customized subscriptions that for example drop the first
```

Comment:
I think we need a bit more context here about how this works, and how its different from normal flow subscriptions.

```suggestion
     * Subscribes to the given Flow within the coroutine scope of the `subscriptionLifecycleOwner`, starting the flow only when the lifecycle is started, and executing with the coroutine context of Mavericks' `subscriptionCoroutineContextOverride`. This can be utilized to create customized subscriptions, for example, to drop the first
```

Reply:
I think we should also add that this is intended to be used with viewmodel.stateflow - are there any issues that arise if this is used with a non mvrx state flow?

Reply:
hmm i think it would still continue to function unless multiple subscriptions end up clashing. added a comment nonetheless explaining intended use